lib/manageiq/environment.rb

```diff
@@ -68,8 +70,12 @@ def self.setup_gemfile_lock
       FileUtils.cp(APP_ROOT.join("Gemfile.lock.release"), APP_ROOT.join("Gemfile.lock"))
     end
 
-    def self.bundle_update(root = APP_ROOT)
-      system!("bundle update --jobs=3", :chdir => root)
+    def self.bundle_update(root = APP_ROOT, force: false)
+      if !force && system("bundle check", [:out, :err] => "/dev/null")
+        puts "== Bundle up to date... Skipping update =="
+      else
+        system!("bundle update --jobs=3", :chdir => root)
+      end
```

Comment:
Can we discuss over voice?  I'm having trouble with keeping all of the scenarios straight, and I feel like something is being missed, perhaps with build?